contracts/tribe/stabilizer/ReserveStabilizer.sol

```diff
@@ -53,7 +53,8 @@ contract ReserveStabilizer is OracleRef, IReserveStabilizer, PCVDeposit {
     function exchangeFei(uint256 feiAmount) public virtual override whenNotPaused returns (uint256 amountOut) {
         updateOracle();
 
-        fei().burnFrom(msg.sender, feiAmount);
+        fei().transferFrom(msg.sender, address(this), feiAmount);
```

Comment:
Did we actually make this change to reserve stabilizers deployed on mainnet? If we didn't, then merging this leaves us in a really weird state because the code in git doesn't match what we have deployed.

Reply:
We did, the only production stabilizer (the TribeReserveStabilizer) uses this